frontend/src/components/pages/Scheduling/DonationInformation.tsx

```diff
@@ -1,31 +1,144 @@
+  Container,
+  FormControl,
+  FormLabel,
+  HStack,
+  Stack,
+  Text,
+} from "@chakra-ui/react";
 import React from "react";
 
+import xl from "../../../assets/donation-sizes/lg.png";
+import lg from "../../../assets/donation-sizes/md.png";
+import md from "../../../assets/donation-sizes/sm.png";
+import sm from "../../../assets/donation-sizes/xs.png";
+import customTheme from "../../../theme";
+import RadioImageSelectGroup from "../../common/RadioImageSelectGroup";
 import SchedulingProgressBar from "../../common/SchedulingProgressBar";
-import { SchedulingStepProps } from "./types";
+import { DonationSizeInterface, SchedulingStepProps } from "./types";
 
-const DonationInformation = ({
+const DonationSizes: DonationSizeInterface[] = [
+  {
+    image: sm,
+    size: "Small",
+    description: "Approximately fills one shelf of the fridge/ pantry",
+  },
+  {
+    image: md,
+    size: "Medium",
+    description: "Approximately fills two shelves of the fridge/ pantry",
+  },
+  {
+    image: lg,
+    size: "Large",
+    description:
+      "Approximately fills four shelves of the fridge/ pantry (full capacity)",
+  },
+  {
+    image: xl,
+    size: "Extra-large",
+    description:
+      "Approximately fills four shelves of the fridge/ pantry (full capacity)",
+  },
+];
+
+const categoriesOptions = [
+  "Dry packaged goods",
+  "Non-perishables",
+  "Fresh produce",
+  "Bread and baked goods",
+  "Oil, spreads, and seasoning",
+  "Tea and coffee",
+  "Frozen meals",
+  "Prepared meals",
+  "Non-alcoholic drinks and juices",
+  "Essential items (masks, hand sanitizer, bags)",
+  "Hygiene products (tampons, pads, soap, etc.)",
+];
+
+const DonationInformation: any = ({
   formValues,
   setForm,
   navigation,
-  isBeingEdited,
 }: SchedulingStepProps) => {
   const { previous, next } = navigation;
+  const { categories, size } = formValues;
+
+  const handleChange = (
+    e: React.ChangeEvent<HTMLInputElement>,
+    name: string,
+  ) => {
+    setForm({ target: { name, value: e } });
+  };
+
+  const handleCheckboxChange = (
+    e: React.ChangeEvent<HTMLInputElement>,
+    item: string,
+  ) => {
+    let categoriesNew = categories;
+    if (e.target.checked) {
+      categoriesNew.push(item);
+    } else {
+      categoriesNew = categories.filter((string) => item !== string);
+    }
+
+    setForm({
+      target: {
+        name: "categories",
+        value: categoriesNew,
+      },
+    });
+  };
 
   return (
-    // Insert Select Date and Time page here
-    <Container>
+    <>
       <SchedulingProgressBar activeStep={1} totalSteps={4} />
-      <Text textStyle="mobileHeader2">This is Donation Information Page </Text>
-      <HStack>
-        <Button onClick={previous} variant="navigation">
-          Back
-        </Button>
-        <Button onClick={next} variant="navigation">
-          Next
-        </Button>
-      </HStack>
-    </Container>
+      <Container px="32px">
+        <Text textStyle="mobileHeader2" mb="20px">
```

Comment:
```suggestion
        <Text textStyle="mobileHeader2" mt="2em">
```
To add spacing between progress bar and Donation Information